composer.json

```diff
@@ -22,7 +22,7 @@
         "ext-mbstring": "*",
         "psr/container": "^1.0|^2.0",
         "yiisoft/arrays": "^2.1",
-        "yiisoft/translator": "^2.0",
+        "yiisoft/translator": "dev-master",
```

Comment:
```suggestion
        "yiisoft/translator": "^2.0.1",
```

Reply:
I think it should stay "^2.0". It will automatically install 2.0.1.